com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/FlowableFacade.java

```diff
@@ -113,6 +113,11 @@ public boolean hasDeadLetterJobs(String processId) {
         return !getDeadLetterJobs(processId).isEmpty();
     }
 
+    private boolean haveAllLeafsHaveDeadLetterJobs(List<Execution> executions) {
```

Comment:
For me an you leaf is an understandable word but not for the rest of the team.
Maybe renaming it to child processes or subProcessLeafs would be better.

Reply:
Done